crates/bitwarden-uniffi/src/lib.rs

```diff
@@ -75,10 +75,15 @@ impl Client {
 }
 
 fn init_logger() {
-    #[cfg(not(target_os = "android"))]
+    #[cfg(not(any(target_os = "android", target_os = "ios")))]
     let _ = env_logger::Builder::from_env(env_logger::Env::default().default_filter_or("info"))
         .try_init();
 
+    #[cfg(target_os = "ios")]
+    let _ = oslog::OsLogger::new("com.8bit.bitwarden")
```

Comment:
:question: Does this need to match the app identifier? Would it be more helpful to use a separate one to differentiate between app logs and SDK logs?

Reply:
Oh I have no idea, good question. I don't think it matters much. Here is the difference when outputing all metadata

![image](https://github.com/bitwarden/sdk/assets/2285588/78d3f7fc-5e95-4d6d-9fb3-f13a856937e2)

Do you have any good suggestions?

Reply:
Hmm I guess it doesn't matter much seeing those logs, the error is already printing `bitwarden_fido::crypto`, which I think is good enough to identify SDK calls.

We can leave it as is then.